openapi/src/common/schemas.yaml

```diff
@@ -172,13 +172,13 @@ components:
       type: object
   # TODO: PF-1218 - when CLI and UI are no longer passing this, disallow it for user-private
   #  resources, make the role List into a single role, and require both fields to be present.
-  #      required: [ userName, privateResourceIamRoles ]
+  #      required: [ userName, privateResourceIamRole ]
```

Comment:
Do the UI or CLI ever include the `PrivateResourceUser` object in the request? If not, is it safe to uncomment this `required` block?

Reply:
Good catch, this should be safe to do now.